### PR TITLE
libusb: fix mingw build

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **libusb/..**

Fix building libusb using mingw on Windows.

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by madebr** [comment]
@uilianris I don't think the errors have anything to do with this pr.

File: recipes/libusb/all/conanfile.py [hunk 1]
```diff
@@ -40,6 +40,11 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    def build_requirements(self):
+        if tools.os_info.is_windows and "CONAN_BASH_PATH" not in os.environ and \
```
Comment (uilianries) [inline recipes/libusb/all/conanfile.py]: @madebr @SSE4 Should we consider it by default in all recipes?

**Comment by uilianries** [comment]
@madebr the problem is that libusb uses system requirements and can't download some packages due EOL (Ubuntu).

It should be related to https://github.com/conan-io/conan-center-index/pull/905

File: recipes/libusb/all/conanfile.py (hunk 1 above)
Comment (madebr) [reply recipes/libusb/all/conanfile.py]: Do you mean removing the `CONAN_BASH_PATH` and `tools.os_info.get_windows_subsytem()` part?
I don't use conan in a msys2 terminal but I suppose some people do.

**Comment by madebr** [comment]
> 
> 
> @madebr the problem is that libusb uses system requirements and can't download some packages due EOL (Ubuntu).
> 
> It should be related to #905

What is the path forward for this pr then?
This would depend on an upgrade of the CI system then?

**Comment by uilianries** [comment]
I just submitted a PR fixing the docker image. It should be enough for this case. THe system_requires problem deserves another PR.

File: recipes/libusb/all/conanfile.py (hunk 1 above)
Comment (danimtb) [reply recipes/libusb/all/conanfile.py]: @uilianries opening an issue to discuss this

Comment (uilianries) [reply recipes/libusb/all/conanfile.py]: done https://github.com/conan-io/conan-center-index/issues/1059

**Pull request merged**